src/megameklab/com/ui/Vehicle/tabs/StructureTab.java

```diff
@@ -38,14 +41,17 @@
 import megamek.common.SuperHeavyTank;
 import megamek.common.Tank;
 import megamek.common.TechConstants;
+import megamek.common.Transporter;
 import megamek.common.TroopSpace;
 import megamek.common.VTOL;
 import megamek.common.verifier.TestEntity;
+import megamek.common.verifier.BayData;
 import megameklab.com.ui.EntitySource;
 import megameklab.com.ui.Vehicle.views.SummaryView;
 import megameklab.com.ui.view.ArmorAllocationView;
 import megameklab.com.ui.view.BasicInfoView;
 import megameklab.com.ui.view.CVChassisView;
+import megameklab.com.ui.view.CVTransportView;

```

Comment:
I don't see CVTransportView defined anywhere, did you forget to push it up?

Reply:
Apparently so.